firmware/thruna/BMS/Test/Src/Test_Faults.cpp

```diff
@@ -53,6 +53,8 @@ FAKE_VALUE_FUNC(EEPROM_StatusTypeDef, read_page, uint16_t, uint8_t, uint8_t *, u
 FAKE_VALUE_FUNC(EEPROM_StatusTypeDef, write_page, uint16_t, uint8_t, uint8_t *, uint16_t);
 FAKE_VALUE_FUNC(EEPROM_StatusTypeDef, page_erase, uint16_t);
 
+static float saved_soc = 100.0f;
```

Comment:
do you need this variable?

Reply:
There needs to be a value input into App_SocStats_Create, for now all tests use SERIES_ELEMENT_FULL_CHARGE_C (100% SOC), but may be changed in the future. I'll replace it with a constant for now.